apps/vivaldi/vivaldi.py

```diff
@@ -53,6 +53,12 @@ def tab_close_wrapper():
     def tab_jump(number: int):
         actions.key(f"ctrl-{number}")
 
+    def command_search(command: str = ""):
+        actions.user.vivaldi_toggle_quick_commands()
```

Comment:
I would recommend you remove this separate action as it is not used in any other places.

Reply:
Are you suggesting that I remove the vivaldi_toggle_quick_commands action? I notice that it is being used [here too](https://github.com/talonhub/community/blob/e781ba8c81042e027f12ced8e8d87b685a1031a5/apps/vivaldi/vivaldi.py#L29). I don't quite understand the suggestion, though I appreciate the feedback.

Reply:
Since that action is only used in this file, I think it makes sense to replace that usage with a call to this action.

Reply:
Can you give me exact code? I don't have this application so I don't want to make a mistake since I can't test it myself-thank you!

Reply:
Oh, I get it. You mean the new "please" action. That makes sense.

Reply:
I think I addressed this - @nriley  does that look good now? Let me know if there's more to do here. Thanks!

Reply:
Looks good. Note that you can click the "chasing arrows" icon next to a reviewer if you want to request a re-review.